### PR TITLE
Add VirtualBox version 6.1.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20210)

### CODE REVIEW

File: manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: Oracle.VirtualBox
+PackageVersion: 6.1.2
+PackageName: VirtualBox
```
Comment (ItzLevvie) [inline manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml:3]:
```suggestion
PackageName: Oracle VM VirtualBox
```

File: manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Publisher: Oracle
```
Comment (ItzLevvie) [inline manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml:4]:
```suggestion
Publisher: Oracle Corporation
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: Oracle.VirtualBox
+PackageVersion: 6.1.2
+PackageName: Oracle VM VirtualBox
+Publisher: Oracle Corporation
+License: GPLv2
+Moniker: virtualbox
+Tags:
+- vm
+- virtual machine
+ShortDescription: Oracle.VirtualBox
```
Comment (ImJoakim) [inline manifests/o/Oracle/VirtualBox/6.1.2/Oracle.VirtualBox.yaml:10]:
```
ShortDescription: Oracle.VirtualBox
```
Can you please add a ShortDescription 😃

**Comment by ImJoakim** [comment]
Please take a look at my comment above 😃 

- https://github.com/microsoft/winget-pkgs/pull/20210#discussion_r666288345

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/20210#pullrequestreview-702193483

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie @ImJoakim Done.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**